README.md

```diff
@@ -17,11 +17,13 @@ since 2011, and has grown to encompass tens of thousands of MySQL nodes.
 
 For more about Vitess, please visit [vitess.io](http://vitess.io).
 
+Vitess has a growing community. You can view the list of adopters [here](ADOPTERS.md).
```

Comment:
A technical comment here: This should be an absolute link.

Our "base" image copies the content of this file and shows it on Docker Hub: https://hub.docker.com/r/vitess/base/
The relative link there won't work.